src/yang/ietf-bgp-common.yang

```diff
@@ -121,6 +121,15 @@ submodule ietf-bgp-common {
         "RFC 4271, Section 4.2.
          RFC 4271, Section 10.";
     }
+    leaf negotiated-hold-time {
+      type uint16;
+      config false;
+      description
+        "The negotiated hold-time for the BGP session";
+      reference
+        "RFC 4271, Section 4.2.
```

Comment:
If we agree on the pattern for the reference statement, described in issue #263, can these statements be updated to reflect the pattern?

Reply:
Did this and a few more since I was in there.